waku/waku_lightpush/client.nim

```diff
@@ -57,29 +46,31 @@ proc sendPushRequest(
     buffer = await connection.readLp(DefaultMaxRpcSize.int)
   except LPStreamRemoteClosedError:
     error "Failed to read responose from peer", error = getCurrentExceptionMsg()
-    if wl.reputationManager.isSome:
-      wl.reputationManager.get().setReputation(peer.peerId, some(false))
+    if wl.peerManager.reputationManager.isSome:
+      wl.peerManager.reputationManager.get().setReputation(peer.peerId, some(false))
```

Comment:
Fine for now but in the future if we decide to have a standalone `reputationManager` we could use `AsyncQueue` as a communication channel.